cypress/support/helpers/runIfToggleEnabled.js

```diff
@@ -0,0 +1,11 @@
+export default ({ service, toggleName, testContext }) => {
+  if (service === 'scotland') return;
```

Comment:
We really need to clean this up! `archive`, `scotland` and probably `sport` need to get in the 🗑️